content/docs/beyond-the-basics/compliance-policies/index.md

```diff
@@ -4,11 +4,371 @@
+    -d "@/path/to/jsonfile" \
+    https://app.rkvst.io/archivist/v1/compliance_policies
+```
+{{< /tab >}}}
+{{< /tabs >}}
+
+4. ***COMPLIANCE_DYNAMIC_TOLERANCE:*** checks if time between correlated events does not exceed defined variability.
+
+To correlate Events, define the attribute `arc_correlation_value` in the Event Attributes and set it to the same value on each pair of events that are to be associated.
+
+For example, "the time between a Maintanence Request Event and Maintanence Performed Event in the last week does not exceed a variation of 0.5 standard deviations around the mean".
+
+The `dynamic_window` is the time period to evaluate on, in this case, one week. The `dynamic_variability` is the number of standard deviations from the mean allowed, in this case, 0.5.
+
+{{< tabs name="compliance_dynamic_tolerance" >}}
+{{{< tab name="UI" >}}
+This type of compliance policy is not yet available for creation via the UI. Check out our YAML or JSON format options!
+{{< /tab >}}
+{{< tab name="YAML" >}}
+```yaml
+---
+steps:
+  - step:
+      action: COMPLIANCE_POLICIES_CREATE
+      description: Create COMPLIANCE_DYNAMIC_TOLERANCE policy
+      print_response: true
+    description: Average time between Maintenance Requested/Performed
+    display_name: Outlying Maintenance Requests
+    compliance_type: COMPLIANCE_DYNAMIC_TOLERANCE
+    asset_filter:
+      - or: [ "attributes.arc_home_location_identity=locations/<location-id>" ]
+    event_display_type: Maintenance Requests
+    closing_event_display_type: Maintenance Performed
+    dynamic_window: "604800"
+    dynamic_variability: "0.5"
+```
+Use the [archivist_runner](https://python.rkvst.com/runner/index.html) command to run your YAML file!
+ 
+```bash
+$ archivist_runner \
+      -u https://app.rkvst.io \
+      --client-id <your-client-id> \
+      --client-secret <your-client-secret> \
+      <path-to-yaml-file>
+```
+{{< /tab >}}
+{{< tab name="JSON" >}}
+```json
+{
+    "compliance_type": "COMPLIANCE_DYNAMIC_TOLERANCE",
+    "description": "Average time between Maintenance Requested/Performed",
+    "display_name": "Outlying Maintenance Requests",
+    "asset_filter": [
+        { "or": ["attributes.arc_home_location_identity=locations/<location-id>"]}
+    ],
+    "event_display_type": "Maintenance Requests",
+    "closing_event_display_type": "Maintenance Performed",
+    "dynamic_window": 604800,
+    "dynamic_variability": 0.5
+}
+```
+Use the curl command to run your JSON file! See instructions for [creating your `BEARER_TOKEN_FILE`](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/) here.
+ 
+```bash
+curl -v -X POST \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    -d "@/path/to/jsonfile" \
+    https://app.rkvst.io/archivist/v1/compliance_policies
+```
+{{< /tab >}}}
+{{< /tabs >}}
+
+5. ***COMPLIANCE_RICHNESS:*** checks if the assertions conducted on an attribute value pass.
```

Comment:
This description is a little opaque and in a different style to the others. 

Might be better to say something like “Checks whether Attributes are within expected bounds or otherwise meet defined conditions “